kwanghwi/myapp/db/migrations/20221220062647_create_posts_table.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE posts (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  content VARCHAR(1000) NULL,
+  content_image VARCHAR(1000) NULL,
+  user_id INT NOT NULL,
+  CONSTRAINT posts_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id),
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
```

Comment:
created_at 컬럼은 데이터가 생성되는 시점의 시간을 저장할 수 있도록,
updated_at 컬럼은 데이터가 수정되는 시점의 시간을 저장할 수 있도록 다음과 같이 수정해주세요!

```suggestion
  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP
```